### DISCUSSION TITLE
Error openDoc error:{"code":5,"parameter":""}

### DISCUSSION THREAD

**Discussion by qlikwebisr** [ptarmiganlabs/butler-cw#272] [General]
On the activation of the script, I'm getting the error of opening the application with the message 'Error openDoc error: {"code": 5, "parameter": ""}'. What could be the reason for such an error? Other applications are opening without a problem. It's a relatively small application with a size of 11.3 MB

**Comment by mountaindude** [comment]
Impossible to say what caused this, more info needed.

For example:

- Is the app you try to open special in some way? Section access etc?
- What does your configuration files look like (remove sensitive info before sharing them!)?
- What Sense version are you using?
- Which Butler CW version are you using?

**Comment by qlikwebisr** [comment]
Thank you for the response.
The Butler CW version is 4.0.2, Qlik Sense Aug 2023,
We disabled the Section access and still app was not loaded.
The settings that we have:
production.yaml
![production_yaml](https://github.com/ptarmiganlabs/butler-cw/assets/64928424/697a51b8-3ae1-4b39-96b3-387407d1b97b)
application settings:
![apps yaml_application_settings](https://github.com/ptarmiganlabs/butler-cw/assets/64928424/e03b2fe0-34e4-4734-bab0-91c258f00812)
Other application loaded without a problem.
The main issue that we cat get indication of what causes the error:
error: openDoc error: {"code":5,"parameter":""}
What does code 5 mean?
